spec/controllers/games_controller_spec.rb

```diff
@@ -2,13 +2,23 @@
 require 'pry'
 
 RSpec.describe GamesController, type: :controller do
+  describe 'games#new action' do
+    it 'should login players' do
+      player = FactoryGirl.build(:player)
+      sign_in player
+    end
+
+    it 'should initialize pieces on the board' do
+      @game = FactoryGirl.build(:game)
+      @piece = FactoryGirl.build(:piece)
+    end
```

Comment:
In a controller test. You simulate sending an HTTP request to the controller - and set an `expect` for the response back.

The way the test is currently is - it creates data. But does not send a request or have an expectation.